crates/virtual-dom-rs/src/patch/apply_patches.rs

```diff
@@ -112,6 +124,10 @@ fn find_nodes(
                 // then it was a delimiter created by virtual-dom-rs in order to ensure that two
                 // neighboring text nodes did not get merged into one by the browser. So we skip
                 // over this virtual-dom-rs generated comment node.
+                // TODO
+                if nodes_to_find.get(&cur_node_idx).is_some() {
```

Comment:
To answer your question about handling `<!-- ptns -->`

This block would check the node's comment and just ignore it if it was `ptns`. Otherwise we'd process it and increment the current node index like we do in the block for `Node::TEXT_NODE` right above this